src/Functions/snowflake.cpp

```diff
@@ -96,54 +100,61 @@ class FunctionSnowflakeToDateTime : public IFunction
             {"value", static_cast<FunctionArgumentDescriptor::TypeValidator>(&isUInt64OrInt64), nullptr, "(U)Int64"}
         };
         FunctionArgumentDescriptors optional_args{
+            {"epoch", static_cast<FunctionArgumentDescriptor::TypeValidator>(&isNativeUInt), isColumnConst, "UInt*"},
```

Comment:
Wouldn't making this epoch the second argument breaks existing queries where the time_zone is the second argument? Wouldn't it be better to have it as third argument?